html/pfappserver/root/src/views/Reports/_components/TheView.vue

```diff
@@ -0,0 +1,242 @@
+<template>
+  <b-card no-body id="card">
+    <b-card-header>
+      <h4 class="mb-0" v-html="id" />
```

Comment:
I suggest to format the id by replacing the `::` by ` / `.

Reply:
@jrouzierinverse mentioned `/` could not be used.

Reply:
I mean formatting for display only: `.split('::').join(' / ')`.